src/toil/lib/accelerators.py

```diff
@@ -103,7 +103,9 @@ def have_working_nvidia_docker_runtime() -> bool:
                 "all",
                 "ubuntu:20.04",
                 "nvidia-smi",
-            ]
+            ],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL
```

Comment:
Why do we suppress these?

Reply:
Because we often expect a failure, and when this fails we get messages printed about `nvidia` not being a Docker runtime that's installed. I was seeing them when running the tests, I think because the new pytest-ier way of doing the tests evaluates the test-gating decorators more/earlier. So I did this to make the tests less noisy.